cmd/git-sync/main.go

```diff
@@ -1043,10 +1053,10 @@ DESCRIPTION
 
     git-sync can pull one time, or on a regular interval.  It can read from the
     HEAD of a branch, from a git tag, or from a specific git hash.  It will only
-    re-pull if the target has changed in the remote repository.  When it
-    re-pulls, it updates the destination directory atomically.  In order to do
-    this, it uses a git worktree in a subdirectory of the --root and flips a
-    symlink.
+	re-pull if the target has changed in the remote repository.  When it
+	re-pulls, it updates the destination directory atomically.  In order to do
+	this, it uses a git worktree in a subdirectory of the --root and flips a
+	symlink.
```

Comment:
Tab?

Reply:
I will have to find some way to auto-prevent that...

Reply:
Go test checking for tabs in said variable?